source/reference/api/reverse.txt

```diff
@@ -44,14 +44,33 @@ To use the ``reverse`` endpoint:
   You cannot update these options after the sync starts.
 - ``mongosync`` must be in the ``COMMITTED`` state.
 - Source and destination clusters must be MongoDB 6.0 or later.
-- :ref:`Unique indexes <index-type-unique>` on the original source
-  cluster must be formatted properly. If an upgraded cluster has unique
-  indexes that were created in MongoDB 4.2 or earlier, you must
-  :ref:`resync <resync-replica-member>` all of the nodes in the
-  original source cluster before reversing.
+- :ref:`index-type-unique` requires proper formatting.  To validate
+  that collection indexes use the proper formatting, see
+  :ref:`c2c-validate-unique-index`.
 - .. include:: /includes/fact-reverse-limitation.rst
 - .. include:: /includes/fact-permissions-body.rst
 
+.. _c2c-validate-unique-index:
+
+Validate Unique Indexes
+~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to reverse direction, ``mongosync`` requires that all
+:ref:`unique <index-type-unique>` indexes use the correct formatting. 
+Clusters that began with MongoDB 4.2 or older and were since
+upgraded may include unique indexes that are not properly formatted.
```

Comment:
I think you should mention that resyncing is still an option. If the client wants to avoid resyncing, then they can run validate.

Reply:
Fixed